app/controllers/tasks_controller.rb

```diff
@@ -18,6 +18,8 @@ def show
   def create
     @task = Task.new(task_params)
 
+    @task.author = current_user
```

Comment:
not sure if I prefer `current_user.tasks.build(task_params)`... one line is better? :P